x-pack/agentbeat/agentbeat.spec.yml

```diff
@@ -314,6 +314,11 @@ inputs:
     platforms: *platforms
     outputs: *outputs
     command: *heartbeat_command
+  - name: autoops_es/metrics
```

Comment:
Move this after the `beat/metrics` below, so that it comes after the definition of the command/outputs/platforms sections you are referencing (and also for consistency).